docs/installation.md

```diff
@@ -1,19 +1,19 @@
-# Installing Percona XtraBackup 8.0
+# Install Percona XtraBackup 8.0
 
 We recommend that you install Percona XtraBackup 8.0 from the official Percona software repositories using the appropriate package manager for your system:
 
-* [Use an APT repo](apt_repo.md)
+* [Use an APT repo](/installation/apt_repo)
 
-* [Use a YUM repo](yum_repo.md)
+* [Use a YUM repo](/installation/yum_repo)
```

Comment:
We need to add a link to [Install with package manager](https://docs.percona.com/percona-xtrabackup/8.0/installation/install-with-package-manager.html)